miio/viomivacuum.py

```diff
@@ -227,96 +223,265 @@ def error(self) -> Optional[str]:
+            "Consumables\n"
+            "===========\n\n"
+            "* Left hours:\n"
+            "  - HYPA filter: {result.hypa_hours_left} hours\n"
+            "  - Main brush: {result.main_brush_hours_left} hours\n"
+            "  - Mop: {result.mop_hours_left} hours\n"
+            "  - Side brush: {result.side_brush_hours_left} hours\n"
+            "* Life left:\n"
+            "  - HYPA filter: {result.hypa_life_left} %\n"
+            "  - Main brush: {result.main_brush_life_left} %\n"
+            "  - Mop: {result.mop_life_left} %\n"
+            "  - Side brush: {result.side_brush_life_left} %\n"
+            "\n"
+            "Map\n"
+            "===\n\n"
+            "Current map ID: {result.current_map_id}\n"
             "Remember map: {result.remember_map}\n"
             "Has map: {result.has_map}\n"
             "Has new map: {result.has_new_map}\n"
-            "Mop mode: {result.mop_mode}\n",
+            "Number of maps: {result.map_number}\n"
+            "\n"
+            "Misc\n"
+            "====\n\n"
+            "Hardware version: {result.hw_info}\n"
+            "Software version: {result.sw_info}\n"
+            "\n"
+            "Unknown properties\n"
+            "=================\n\n"
+            "Light state: {result.light_state}\n"
+            "Working: {result.working}\n"
+            "Charging: {result.charging}\n"
+            "Mop route: {result.mop_route}\n"
+            "Mop type: {result.mop_type}\n"
+            "Order time: {result.order_time}\n"
+            "Start time: {result.start_time}\n"
+            "v_state: {result.v_state}\n"
+            "water_percent: {result.water_percent}\n"
+            "zone_data: {result.zone_data}\n",
         )
     )
     def status(self) -> ViomiVacuumStatus:
         """Retrieve properties."""
         properties = [
-            "run_state",
-            "mode",
-            "err_state",
             "battary_life",
             "box_type",
+            "cur_mapid",
+            "err_state",
+            "has_map",
+            "has_newmap",
+            "hw_info",
+            "hypa_hours",
+            "hypa_life",
+            "is_charge",
+            "is_mop",
+            "is_work",
+            "light_state",
+            "main_brush_hours",
+            "main_brush_life",
+            "map_num",
+            "mode",
+            "mop_hours",
+            "mop_life",
+            "mop_route",
             "mop_type",
-            "s_time",
+            "order_time",
+            "remember_map",
+            "repeat_state",
+            "run_state",
             "s_area",
+            "s_time",
+            "side_brush_hours",
+            "side_brush_life",
+            "start_time",
             "suction_grade",
+            "sw_info",
+            "v_state",
             "water_grade",
-            "remember_map",
-            "has_map",
-            "is_mop",
-            "has_newmap",
+            "water_percent",
+            "zone_data",
```

Comment:
This is quite many variables, are you sure this is working?

Reply:
fixed

Reply:
What was exactly fixed? Did you maybe forgot to push the changes?